CMakeLists.txt

```diff
@@ -532,6 +530,57 @@ while(library_variants_copy)
             ${INSTALL_DIR}/lib/${target} ${install_directory}/lib
     )
 
+    #picolibc tests
+    string(REPLACE " " ";" clang_compile_flags ${runtimes_flags})
+    add_custom_target(check-picolibc_${variant})
+    set(picolibc_test_ld ${picolibc_SOURCE_DIR}/scripts/test-arm.ld)
+    if(${variant} EQUAL "aarch64")
+        set(picolibc_test_ld ${picolibc_SOURCE_DIR}/scripts/test-aarch64.ld)
```

Comment:
Since AArch64 tests don't seem to be working you could try using `samples/ldscripts/raspi3b.ld`. See #169